db/structure.sql

```diff
@@ -64,8 +50,6 @@ CREATE TYPE public.gender_enum AS ENUM (
 
 SET default_tablespace = '';
 
-SET default_with_oids = false;
-
```

Comment:
Can you remove these changes? Not sure why this diff is here, we could debug the postgres differences at some point when pairing.